### PR TITLE
Add task price and execution duration

### PR DESCRIPTION
**Pull request by antho1404**
### API

Added support of task price and execution duration

**Breaking change:** `service.create` requires a price in the definition

### Compiler

Added support of task price

### Orchestrator

Added support for execution duration

**Breaking change:** `execution.create` does not accept a price anymore

### Service

Added support for execution duration

### CODE REVIEW

File: packages/api/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ## [Unreleased](https://github.com/liteflow-labs/liteflow-js/releases/tag/%40liteflow%2Fapi%40X.X.X)
 
 #### Breaking Changes
+
+- [#](https://github.com/liteflow-labs/liteflow-js/pull/) Add required task price field to service and execution price with time
```
Comment (antho1404) [inline packages/api/CHANGELOG.md:7]:
```suggestion
- [#139](https://github.com/liteflow-labs/liteflow-js/pull/139) Add required task price field to service and execution price with time
```

File: packages/compiler/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,9 @@
 
 #### Breaking Changes
 #### Improvements
+
+- [#](https://github.com/liteflow-labs/liteflow-js/pull/) Add support of task price, default set to 0
```
Comment (antho1404) [inline packages/compiler/CHANGELOG.md:8]:
```suggestion
- [#139](https://github.com/liteflow-labs/liteflow-js/pull/139) Add support of task price, default set to 0
```

File: packages/orchestrator/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,9 @@
 ## [Unreleased](https://github.com/liteflow-labs/liteflow-js/releases/tag/%40liteflow%2Forchestrator%40X.X.X)
 
 #### Breaking Changes
+
+- [#](https://github.com/liteflow-labs/liteflow-js/pull/) Remove price from execution create request. Add execution duration support
```
Comment (antho1404) [inline packages/orchestrator/CHANGELOG.md:7]:
```suggestion
- [#139](https://github.com/liteflow-labs/liteflow-js/pull/139) Remove price from execution create request. Add execution duration support
```

File: packages/api/src/execution.ts [hunk 4]
```diff
@@ -30,6 +30,8 @@ export type IExecution = {
     blockHeight: number;
   }[];
   address?: string
+  start?: number;
+  stop?: number;
```
Comment (NicolasMahe) [inline packages/api/src/execution.ts:34]: Are we sure about the `number` type here? We will have issue with overflow.

Comment (antho1404) [reply packages/api/src/execution.ts:34]: good catch, this is returned by amino as a string

Comment (antho1404) [inline packages/api/src/execution.ts:34]:
```suggestion
  start?: string;
  stop?: string;
```

**Pull request merged**